daemon/daemon.go

```diff
@@ -607,7 +595,7 @@ func NewDaemon(c *Config) (*Daemon, error) {
 		consumableCache:   policy.NewConsumableCache(),
 		policy:            policy.NewPolicyRepository(),
 		ignoredContainers: make(map[string]int),
-		buildEndpointChan: make(chan *endpoint.Request, common.EndpointsPerHost),
+		buildEndpointChan: make(chan *endpoint.Request, 512),
```

Comment:
If this is the code then the surrounding code needs to be changed. We can have the same endpoint being rebuilt multiple times. I've bumped endpoints per host to 64K. We absolutely don't want a build backlog of 64K.